analysis/10c_an_interaction_cox_models_vaccine_child_prevacc.do

```diff
@@ -0,0 +1,282 @@
+********************************************************************************
+*
+*	Do-file:		10c_an_interaction_cox_models_vaccine_child_prevacc.do
+*
+*	Project:		Exposure children and COVID risk
+*
+*	Programmed by:	TCowling, based on Harriet's files
+*
+*	Data used:		analysis_dataset.dta
+*
+*	Data created:	None
+*
+*	Other output:	Log file: 10c_an_interaction_cox_models_vaccine_`outcome'_child_prevacc.log
+*
+********************************************************************************
+*
+*	Purpose:		This do-file performs multivariable (fully adjusted) 
+*					Cox models, with an interaction by vaccine and censoring
+*                   when all children aged 12-17 in the household have had
+*                   a vaccine (if they all have done)
```

Comment:
perhaps make it clear here it's when all children in the household had their 2nd vaccination, rather than first?

Reply:
Good spot - have edited!